packages/android_intent/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: android_intent
+description: A plugin for launching Intents on Android.
+version: 0.0.1
+
```

Comment:
This is most likely missing an `authors` and `homepage` entry and possibly an `environment` entry, see https://github.com/flutter/plugins/blob/master/packages/firebase_messaging/pubspec.yaml for an example.

You can also run `pub publish --dry-run` on this plugin to see if `pub` is happy with this pubspec. (use `pub` from `<flutter sdk dir>/bin/cache/dart-sdk/bin/pub`)